css/decanter-grid.css

```diff
@@ -1,3 +1,6 @@
+#kss-node .kss-section.is-fullscreen {
```

Comment:
This markup is for the styleguide only and should not be rendered to the grid library.

Reply:
+1